Automattic/wp-calypso
# Refactor storybook config

#### Changes proposed in this Pull Request
1. Refactor storybook config so that it's reusable.
2. Fix babel config. Previously, it was not transpiling some code. (For some reason, it now wants a target.)
3. Fix emotion 11 resolution. This isn't a problem until #54793, when some components start trying to resolve emotion 1.

#### Testing instructions
Run all of the storybooks locally and ensure that they render components in the browser:

- `yarn composite-checkout:storybook:start`
- `yarn components:storybook:start`
- `yarn search:storybook:start`
